glib/src/log.rs

```diff
@@ -331,17 +331,68 @@ pub fn log_default_handler(log_domain: Option<&str>, log_level: LogLevel, messag
 /// // trailing commas work as well:
 /// g_log!(Some("test"), LogLevel::Warning, "test: {} {}", x, "a",);
 /// ```
+///
+/// To be noted that the log domain is optional:
+///
+/// ```no_run
+/// use glib::{LogLevel, g_log};
+///
+/// // As you can see: no log domain:
+/// g_log!("test", LogLevel::Message, "test");
```

Comment:
You actually provide a log domain here :)

Reply:
Woups! Good catch. :)